travis/empty-make-new-peoject.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e
```

Comment:
Typo in the name of the file "empty-make-new-peoject.sh" => "empty-make-new-project.sh"